src/mousetracker.js

```diff
@@ -1238,8 +1238,9 @@
         },
 
         /**
-         * @function Increment this pointer's contact count.
+         * Increment this pointer's contact count.
```

Comment:
@LarissaSmith Evidently JSDoc doesn't like it if the description is on the same line as the `@function`; it needs to be above, for future reference :)